system/shthreads.py

```diff
@@ -261,9 +261,16 @@ def status(self):
         # False     | not None    | stopped
         # True      |     None    | impossible
         # True      | not None    | running
-        if self.isAlive():
+        
+        # `Thread.isAlive()` was removed in Python 3.9
```

Comment:
`Thread.is_alive()` was added in Python 2.6 so we can use that form everywhere.
* https://docs.python.org/2/library/threading.html#threading.Thread.is_alive